vite.config.mts

```diff
@@ -89,18 +89,20 @@ function getModuleName(id: string): string {
   return fileName.replace(/\.\w+$/, '')  // Remove file extension
 }
 
-const DEV_SERVER_COMFYUI_URL = process.env.DEV_SERVER_COMFYUI_URL || 'http://127.0.0.1:8188'
+/* we keep the old DEV_SERVER_COMFYUI_URL for compatibility with old .env files */
+const VITE_COMFYUI_SERVER_URL = process.env.VITE_COMFYUI_SERVER_URL || (process.env.DEV_SERVER_COMFYUI_URL || 'http://127.0.0.1:8188')
```

Comment:
This is wrong - "DEV_SERVER" refers to *not* the vite server, it refers to the ComfyUI python (API-provider) server